paddle/capi/error.h

```diff
@@ -27,4 +27,21 @@ typedef enum {
   kPD_UNDEFINED_ERROR = -1,
 } paddle_error;
 
+static const char* paddle_error_string(paddle_error err) {
```

Comment:
Move the function implementation into a .cc file and avoid generating duplicate copies of code in the inference library.

Reply:
Done.